spec/factories/groupings.rb

```diff
@@ -1,6 +1,14 @@
 FactoryGirl.define do
+
   factory :grouping do
     association :group
     association :assignment
   end
+
+factory :grouping2, class: Grouping do
+    association :group
+    association :assignment
+    inviter { Student.new(section: Section.new) }
+  end
```

Comment:
`end` at 12, 2 is not aligned with `factory :grouping2, class: Grouping do` at 8, 0